EIPS/eip-3074.md

```diff
@@ -0,0 +1,179 @@
+---
+eip: 3074
+title: Sponsored Transaction Precompile
+author: Sam Wilson (@SamWilsn)
+discussions-to: <URL>
```

Comment:
This is required for draft EIPs.  We recommend a forum post at https://ethereum-magicians.org/, but you can also create a GitHub issue for discussion if you prefer.  You cannot use this PR as discussion-to URL.